JumpscaleCore/clients/tests/test_sshkey_client.py

```diff
@@ -19,33 +19,36 @@ def tearDown(self):
         self.delete_from_sshdir()
         self.delete_client_method(self.sshkey_client, "jumpscale.sshkey.client", self.sshkeyclient_name)
 
-    def delete_from_sshdir(self):
-        self.info("Check delete_from_sshdir method in sshkey client, which delete ssh key files from ssh directory")
-        self.sshkey_client.delete_from_sshdir()
-        self.info("Check that sshkey files are deleted from ssh directory")
-        if not (
-            os.path.isfile("{}/{}.pub".format(self.sshkey_dir, self.sshkeyclient_name))
-            and os.path.isfile("{}/{}".format(self.sshkey_dir, self.sshkeyclient_name))
-        ):
-            return True
-        else:
-            return False
-
-    def test001_load_from_filesystem(self):
+    # def delete_ssh_key_from_ssh_directory(self):
+    #     self.info("Check delete ssh key files from ssh directory using delete_from_sshdir method")
+    #     self.sshkey_client.delete_from_sshdir()
+    #     if not (
+    #         os.path.isfile("{}/{}.pub".format(self.sshkey_dir, self.sshkeyclient_name))
+    #         and os.path.isfile("{}/{}".format(self.sshkey_dir, self.sshkeyclient_name))
+    #     ):
+    #         return True
+    #     else:
+    #         return False
+
+    def test001_load_sshkey_client_into_database(self):
         """
         TC 469
-        Test load_from_filesyatem method which load sshkey files (public and private) from filesystem to database bcdb.
+        Test to load sshkey client into database.
 
         **Test scenario**
-        #. Use delete first to delete the client from database and check for the existence of it (it should be removed).
-        #. Use load_from_filesystem method to load sshkey files from filesystem to database.
-        #. Check the existence of the client in BCDB.
+        #. Create sshkey client.
+        #. Check the existence of the client in database, should be there.
```

Comment:
`Check the existence of the client in database, should be there.`
where is this check in the test cases?!!